src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -81,11 +74,6 @@ def load_command_table(self, _):
         g.command('start', 'begin_start', supports_no_wait=True)
         g.custom_command('get-os-options', 'aks_get_os_options')
 
-    # AKS container service commands
-    with self.command_group('aks', container_services_sdk, client_factory=cf_container_services) as g:
```

Comment:
why remove these.

Reply:
I suppose the implementation of this command in aks-preview is duplicated with the corresponding commands in azure-cli. The command depends on the container_services operations provided by an old version 2017-07-01 client. That is to say, the implementation in aks-preview does not depend on the new api version, nor does it have any functional expansion.

After removing the command declaration here, if the user uses this command after installing aks-preview, it will fall back to the implementation in azure-cli, which makes no difference to the user.

Reply:
And we have a live test case to test this command, the [test result](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=48702401&view=logs&j=8a3f31fe-3b60-55c6-ea75-9b9b79642975&t=fd897544-4eec-5300-ac62-96c3f33aa52e&l=816) looks all good.